src/server/game.js

```diff
@@ -82,15 +82,32 @@ export class Game {
     Object.values(round).forEach(({ player, prompt }) => {
       player.setPrompt(prompt);
     });
+
+    // start a 30 second timer
+    let time = 30;
+    this.timer = setInterval(() => {
+      time -= 1;
+
+      this.players.forEach((player) => {
+        player.setPrompt(time);
```

Comment:
Here we'd be overriding the unique prompt stored in the player instance, which would cause issues with reloads. I think we should be setting a new piece of state for the timer, with its own setter method (similar to how `setViewDrawing` is separate)